lib/custom_navigation_bar.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:mood_tracker/features/add_new_note/presentation/screens/add_new_note_screen.dart';
+import 'package:mood_tracker/features/settings_screen/settings_screen.dart';
+
+class CustomNavigationBar extends StatefulWidget {
+  const CustomNavigationBar({Key? key}) : super(key: key);
+
+  @override
+  State<CustomNavigationBar> createState() => _CustomNavigationBarState();
+}
+
+class _CustomNavigationBarState extends State<CustomNavigationBar> {
+  int selectedIndex = 0;
+  static const TextStyle optionStyle =
+      TextStyle(fontSize: 30, fontWeight: FontWeight.bold);
+  static const List<Widget> _widgetOptions = <Widget>[
+    Text(
+      'Index 0: Calendar',
+      style: optionStyle,
+    ),
+    Text(
+      'Index 1: Server',
+      style: optionStyle,
+    ),
+    Text(
+      'Index 2: BarGraph',
+      style: optionStyle,
+    ),
+    SettingsScreen(),
+  ];
+  void _onItemTapped(int index) {
+    setState(() {
+      selectedIndex = index;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: _widgetOptions[selectedIndex],
+      floatingActionButton: Container(
+        decoration: BoxDecoration(
+          color: Colors.transparent,
+          borderRadius: const BorderRadius.all(
+            Radius.circular(50),
+          ),
+          boxShadow: [
+            BoxShadow(
+              color: const Color(0xFFFF7562).withOpacity(0.3),
+              spreadRadius: 7,
+              blurRadius: 10,
+              offset: const Offset(0, 0),
+            ),
+          ],
+        ),
+        child: FloatingActionButton(
+          elevation: 0.0,
+          onPressed: () {
+            Navigator.of(context).push(
+              MaterialPageRoute(
+                builder: (context) => const AddNewNoteScreen(),
+              ),
+            );
+          },
+          backgroundColor: const Color(0xFFFF7562),
+          child: SvgPicture.asset('images/Human.svg'),
+        ),
+      ),
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
+      bottomNavigationBar: BottomAppBar(
+        elevation: 100,
+        shape: const CircularNotchedRectangle(),
+        notchMargin: 16,
+        child: SizedBox(
+          height: 56.0,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              TextButton(
```

Comment:
there is no need to use TextButton. InkWell is enough for now. Then it'll be refactored to custom widget with appropriate tap state, etc.